### ISSUE TITLE
Getting Started examples need "0dbfs = 1" 

### ISSUE THREAD

**Issue by DrEntropy** [csound/csound.github.io#84]
The "full examples" on https://csound.com/get-started.html  will produce silence if the reader doesn't include the at least the header line:
`0dbfs = 1`
This can be frustrating for someone who is just trying to 'get started'.  If they just used the example they would only get silence and probably have no idea why.
I recommend somewhere after describing the header mentioning that it needs to be included in any examples on this page, and then a reminder for the first "full example" , perhaps just including the header in that example. 

In other words, the first full example would become:
``` 
<CsoundSynthesizer>
<CsInstruments>
sr = 44100
nchnls = 2
0dbfs = 1

instr 1
aOut vco2 1, p4
out aOut
endin

</CsInstruments>
<CsScore>
i1 0 1 100
i1 1 1 200
i1 2 1 300
</CsScore>
</CsoundSynthesizer>
```

**Comment by vlazzarini** [comment]
Or the examples could be modified to refer to Odbfs. 

a1 vco2 0dbfs,p4

etc

**Comment by joshkopecek** [comment]
Seconding this. If it's as simple as editing the markdown, would you accept PRs?

**Comment by joachimheintz** [comment]
if have also added the 0dbfs=1 statement in other parts of this site.
